roles/install/tasks/detect_source.yml

```diff
@@ -0,0 +1,23 @@
+---
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2022, Felix Fontein
+
+- name: Check whether system packages are a valid source of sops {{ sops_version }}
+  when: _community_sops_install_system_has_system and (sops_version == 'latest' or not _community_sops_install_system_has_system_latest_only)
```

Comment:
```suggestion
  when:
    - _community_sops_install_system_has_system
    - sops_version == 'latest' or not _community_sops_install_system_has_system_latest_only
```

is more readable.

Reply:
Implemented in dfc2a92b45841ac94e5d1fbf5542303664562a50.